common/exception/ErrorMessage.java

```diff
@@ -42,7 +42,7 @@ public static class Server extends ErrorMessage {
         public static final Server EXITED_WITH_ERROR =
                 new Server(3, "Exited with error.");
         public static final Server UNCAUGHT_EXCEPTION =
-                new Server(4, "Uncaught exception thrown at thread '%s'.");
```

Comment:
we were trying to add the exception before but the string formatting would've not matched the second argument